### PR TITLE
MAINTAINERS: remove @zlatanvasovic from organization owners

### LINKED ISSUE #7538: Remove organization owner: @zlatanvasovic
I am removing myself from the organization as I've been inactive in the past and I'm moving onto different things.

Thanks for collaborating on this amazing project, and best of luck in the future! 🎉 🎉 🎉

### PR DESCRIPTION
**Pull request by zlatanvasovic**
Close #7538

### CODE REVIEW

**Comment by zlatanvasovic** [comment]
Successfully removed myself, you can approve it 😎

File: MAINTAINERS.md [hunk 1]
```diff
@@ -159,3 +157,5 @@ An automated list can be found [here](https://github.com/orgs/tldr-pages/people)
   until [18 January 2018](https://github.com/tldr-pages/tldr/issues/1878#issuecomment-358610454)
 - Waldir Pimenta ([@waldyrious](https://github.com/waldyrious)):
   until [26 August 2018](https://github.com/tldr-pages/tldr/issues/2257)
+- Zlatan Vasović ([@zlatanvasovic](https://github.com/zlatanvasovic)):
+  [18 June 2020](https://github.com/tldr-pages/tldr/issues/4113) — [14 December 2021](https://github.com/tldr-pages/tldr/issues/7538)
```
Comment (navarroaxel) [inline MAINTAINERS.md:161]: Looks like this is the correct format
```suggestion
  until [14 December 2021](https://github.com/tldr-pages/tldr/issues/7538)
```

Comment (zlatanvasovic) [reply MAINTAINERS.md:161]: I believe that was used for original organization owners, before the introduction of the maintainers system. This way there is no information when did I become an organization owner, but I'm fine with both formats.

**Review by mfrw — APPROVED** [review]
I am of the opinion that we should stick with the current format. So +1 for @navarroaxel's suggestion.
In any case; thank you @zlatanvasovic for your contributions and Good Luck for your future endeavors. :)

**Comment by zlatanvasovic** [comment]
Whatever you decide is fine. @navarroaxel @mfrw you can just edit it and merge, if you want to speed up the process. :smile:

**Comment by zlatanvasovic** [comment]
Actually, you **have** to edit it, because I don't have access to the repository anymore, and this branch is in the repository, not my fork. 😆

**Comment by navarroaxel** [comment]
@zlatanvasovic oh, you can keep your collaborator (write) access if you want. Please confirm.

File: MAINTAINERS.md (hunk 1 above)
Comment (sbrl) [reply MAINTAINERS.md:161]: I think @navarroaxel's suggestion is right. But I don't understand what @zlatanvasovic is saying fully.

Comment (zlatanvasovic) [reply MAINTAINERS.md:161]: It's fine, you can accept his suggestion. :smile:

**Comment by zlatanvasovic** [comment]
@navarroaxel I'd rather not keep it, don't worry :D

**Review by zlatanvasovic — COMMENTED** [review]
You definitely can merge it now, looks good!

**Review by navarroaxel — APPROVED** [review]

**Review by CleanMachine1 — APPROVED** [review]

**Comment by zlatanvasovic** [comment]
@navarroaxel ready for merge?

**Pull request merged**